GeneralsMD/Code/Libraries/Source/WWVegas/WWAudio/SoundScene.h

```diff
@@ -178,20 +195,7 @@ class SoundSceneClass
 		//////////////////////////////////////////////////////////////////////
 		//	Collection methods
 		//////////////////////////////////////////////////////////////////////		
-		class AudibleInfoClass : public MultiListObjectClass, public AutoPoolClass<AudibleInfoClass, 64>
```

Comment:
This is about visibility. Do not move. Just add `public` before this class and `protected` behind this class.